src/pharmpy/internals/math.py

```diff
@@ -105,10 +106,18 @@ def se_delta_method(expr, values, cov):
     return se
 
 
-def is_positive_semidefinite(A):
+def is_symmetric(A):
+    return (A == A.T).all().all()
+
+
+def is_positive_semidefinite(A, eigvals=None):
```

Comment:
Having the function as argument here creates a coupling. Perhaps a different function for known symmetric matrices?